src/components/organisms/NavigatorPane/CompareButton.tsx

```diff
@@ -7,13 +7,22 @@ import {SwapOutlined} from '@ant-design/icons';
 import {TOOLTIP_DELAY} from '@constants/constants';
 
 import {useAppDispatch} from '@redux/hooks';
-import {compareToggled} from '@redux/reducers/compare';
+import {ComparisonView, compareToggled} from '@redux/reducers/compare';
 
 export const CompareButton: React.FC = ({children}) => {
   const dispatch = useAppDispatch();
 
   const onClickClusterComparison = () => {
-    dispatch(compareToggled(true));
+    const defaultView: ComparisonView = {
+      leftSet: {
+        type: 'local',
+      },
+      rightSet: {
+        type: 'cluster',
+        context: 'minikube',
+      },
+    };
+    dispatch(compareToggled({value: true, initialView: defaultView}));
```

Comment:
The `initialView` seemed like a nice small feature that opens a lot of possibilities. Imagine you are previewing a Helm chart, on the other _values.yaml_ files we could on right-click add a compare that opens this view pre-filled.

This defaultView here will be removed later but is now for faster development feedback cycles.